var/spack/repos/builtin/packages/py-morphio/package.py

```diff
@@ -10,7 +10,7 @@ class PyMorphio(PythonPackage):
     """Python library for reading / writing morphology files"""
 
     homepage = "https://github.com/BlueBrain/MorphIO/"
-    url      = "https://github.com/BlueBrain/MorphIO.git"
+    git      = "https://github.com/BlueBrain/MorphIO.git"
```

Comment:
Is `git` now recognised? In the version we had to put `git=url` or `git=git` in this case.

Reply:
Out of curiosity, what's the effect of this change? ('RTFM' counts as an answer :) ).

Reply:
`git` seems to be recognized now - at least locally for me.

@arsenius7 there's some implicit semantics going on when specifying versions. `tag` in a `version` command requires `git` to be defined.

Reply:
`git=` is recognized and can be used standalone (no `url=`) since over 1y ago. We had this discussion some time ago for neurodamus packages :)

Reply:
my bad! 

> If a package contains both a url and git class-level attribute, Spack decides which to use based on the arguments to the version() directive. Versions containing a specific branch, tag, or revision are assumed to be for VCS download methods, while versions containing a checksum are assumed to be for URL download methods.

[docs](https://spack.readthedocs.io/en/latest/packaging_guide.html#fetching-from-code-repositories).